posts/2025-03-25-25.0.0.3.adoc

```diff
@@ -1,91 +1,30 @@
 ---
 layout: post
-title: "TITLE"
```

Comment:
@navaneethsnair1 - this commit has most of the initial edits I made from the templated content